itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/universal/methods/AbstractQuarkusTest.java

```diff
@@ -156,6 +156,37 @@ public static void refreshProject(String projectName, String projectType) {
 			new ContextMenuItem(TextLabels.GRADLE_CONTEXT_MENU_ITEM, TextLabels.REFRESH_GRADLE_PROJECT).select();
 		}
 		new WaitWhile(new JobIsRunning(), TimePeriod.LONG);
+		if (projectType.equals(TextLabels.MAVEN_TYPE)) {
+			refreshProjectWorkaround(projectName);
+		}
+	}
+	
+	private static void refreshProjectWorkaround(String projectName) {
+		try {
+			Thread.sleep(10000);
+		} catch (InterruptedException e) {
+							// CRS need some time for
+							// check errors again
+			e.printStackTrace();
+		}
+		
+		ProblemsView problemsView = new ProblemsView();
+		problemsView.open();
+		List<Problem> problems = problemsView.getProblems(ProblemType.ERROR);
+		assertEquals("There should be 1 error in imported project", 1, problems.size());
```

Comment:
Always better to check for specific error in problems view.

Reply:
done